### PR TITLE
Fix reply visibility not inherit issue

### PR DESCRIPTION
**Pull request by MainasuK**
- Fix the reply visibility not inherit issue
- Add throttle to auto-complete query 
- Restore keyboard shortcuts for the post compose scene
- Restore post compose limit 
- Update i18n resources

### CODE REVIEW

**Review by zeitschlag — COMMENTED** [review]
I was a bit confused to see so many l10n-files in a PR whose title states something about reply visibility. But that's it :)

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Attachment/AttachmentView.swift [hunk 1]
```diff
@@ -123,7 +123,7 @@ public struct AttachmentView: View {
                                 case .remove:
                                     switch viewModel.uploadState {
                                     case .compressing:
-                                        return "Comporessing..."    // TODO: i18n
+                                        return "Compressing..."    // TODO: i18n
```
[the remark this answers is not in the document]
Comment (MainasuK) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Attachment/AttachmentView.swift:126]: That's the defect of the current i18n workflow. The word just added into the `JSON` and `stringsdict` files cannot be sync until merged. 

I just try to use the source template file to create the `en.lproj` resource directly. (https://github.com/mastodon/mastodon-ios/issues/539) 

So we can generate the `L10n` with a new string just added. But that's will give a string likes "Scene.Compose.Attachment.Comporessing" in a non-English language. I'm not sure that is an improvement. What do you think.

[the remark this answers is not in the document]
Comment (MainasuK) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Attachment/AttachmentView.swift:126]: That's the defect of the current i18n workflow. The word just added into the `JSON` and `stringsdict` files cannot be sync until merged. 

I just try to use the source template file to create the `en.lproj` resource directly. (https://github.com/mastodon/mastodon-ios/issues/539) 

So we can generate the `L10n` with a new string just added. But that's will give a string likes "Scene.Compose.Attachment.Comporessing" in a non-English language. I'm not sure that is an improvement. What do you think.

File: Mastodon/Scene/Compose/ComposeViewController.swift [hunk 2]
```diff
@@ -44,18 +44,20 @@ final class ComposeViewController: UIViewController, NeedsDependency {
...
+    //     label.font = .systemFont(ofSize: 15, weight: .regular)
+    //     label.text = "500"
+    //     label.textColor = Asset.Colors.Label.secondary.color
+    //     label.accessibilityLabel = L10n.A11y.Plural.Count.inputLimitRemains(500)
+    //     return label
+    // }()
+    // private(set) lazy var characterCountBarButtonItem: UIBarButtonItem = {
+    //     let barButtonItem = UIBarButtonItem(customView: characterCountLabel)
+    //     return barButtonItem
+    // }()
```
[the remark this answers is not in the document]
Comment (MainasuK) [reply Mastodon/Scene/Compose/ComposeViewController.swift]: You're right. Removed.

File: MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/ComposeContentViewModel.swift [hunk 3]
```diff
@@ -453,7 +479,7 @@ extension ComposeContentViewModel {
         public var errorDescription: String? {
             switch self {
             case .pollHasEmptyOption:
-                return "The post poll is invalid"  // TODO: i18n
+                return "The poll is invalid"  // TODO: i18n
```
[the remark this answers is not in the document]
Comment (MainasuK) [reply MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/ComposeContentViewModel.swift]: The new i18n workflow prevent it now.

**Pull request merged**